### PR TITLE
Browserify

### PR DESCRIPTION
**Pull request by KevinGlinski**
wanted to get the platform sdk in line with the client apps sdk.  

this sdk usage 
```
var users = new purecloud.platform.UsersApi(session);
users.getMe()
```

then the client apps
```
purecloud.apps.users.showProfile(profileId);
```

slightly different still in that client apps are all static methods where the platform sdk is in the context of a session, but I think that is ok.

### CODE REVIEW

**Comment by KevinGlinski** [comment]
@andrejewski @justinaray

File: gulpfile.js [hunk 1]
```diff
@@ -198,20 +206,22 @@ function parseJsonSchema(opts, type){
...
+                .pipe(rename({extname: '.min.js'}))
+                .pipe(gulp.dest('./dist/'));
 };
 
 var buildNode = function() {
-    return gulp.src('./gen/*.js')
-                .pipe(addsrc.prepend('./templates/node_pre.js'))
-                .pipe(addsrc.append('./nodegen/*.js'))
+    return gulp.src('./gen/index.js')
+                //.pipe(addsrc.prepend('./templates/node_pre.js'))
```
Comment (KevinGlinski) [inline gulpfile.js]: i'll clean up these 2 lines

File: package.json [hunk 2]
```diff
@@ -21,24 +27,30 @@
...
-    "gulp-replace": "*",
-    "gulp-rimraf": "^0.1.1",
-    "jasmine": "^2.4.1",
-    "jsdoc": "~3.3.3",
-    "lodash": "4.0.0",
-    "mustache": "~2.2.0",
-    "openapi-model-example": "git://github.com/KevinGlinski/openapi-model-example.git",
-    "purecloud-api-sdk-common": "git://github.com/MyPureCloud/purecloud_api_sdk_common.git",
-    "run-sequence": "1.1.5"
+      "babel-preset-es2015": "^6.6.0",
```
Comment (justinaray) [inline package.json]: I think you're missing a few deps in here: `gutil`, `jshint`, `pre-push`.  Not sure if you're using `watchify`.

**Comment by justinaray** [comment]
Aside from the missing deps above, this LGtM.

I successfully built and ran the examples locally.

**Pull request merged**